### PR TITLE
Add JSON C3

### PR DESCRIPTION
**Pull request by ttytm**
I'd add an bfk bench in a separate PR if an initial working bench for the language has been added.

### CODE REVIEW

File: common/commands.mk [hunk 1]
```diff
@@ -9,6 +9,7 @@ V_FLAGS := -prod -no-bounds-checking -prealloc -cflags "-Wno-error=implicit-func
 V_VSL_CBLAS_FLAGS := $(V_FLAGS) -d cblas
 ZIG_FLAGS := -O ReleaseFast
 
+C3_BUILD =	c3c -05 --safe=no $^
```
Comment (ttytm) [inline common/commands.mk:12]: Regarding the --safe flag: it's to disable bounds check, similar as it's done for other languages.

Ref.:
```
--safe=<yes|no>            - Turn safety (contracts, runtime bounds checking, null pointer checks etc) on or off.
```

Comment (nuald) [reply common/commands.mk:12]: Yes, that's expected to be set, thanks for the clarification.

**Comment by nuald** [comment]
Follow-up has published with the results and few minor issues fixes.

**Comment by ttytm** [comment]
> Follow-up has published with the results and few minor issues fixes.

Very nice thank you!

**Pull request merged**